_episodes/03-skimage-images.md

```diff
@@ -5,18 +5,20 @@ exercises: 50
 questions:
 - "How can the skimage Python computer vision library be used to work with images?"
 objectives:
-- "Read, display, and save images."
+- "Read and save images with imageio."
+- "Display images with matplotlib."
 - "Resize images with skimage."
 - "Perform simple image thresholding with NumPy array operations."
 - "Extract sub-images using array slicing."
 keypoints:
-- "Images are read from disk with the `skimage.io.imread()` function."
+- "Images are read from disk with the `iio.imread()` function."
 - "We create a window that automatically scales the displayed image
 with matplotlib and calling `show()` on the global figure object."
-- "colour images can be transformed to grayscale using `skimage.color.rgb2gray()` or
-be read as grayscale directly by passing the argument `as_gray=True` to `skimage.io.imread()`."
+- "Colour images can be transformed to grayscale using `skimage.color.rgb2gray()` or,
+in many cases,
+be read as grayscale directly by passing the argument `mode="L"` to `iio.imread()`."
```

Comment:
```suggestion
be read as grayscale directly by passing the argument `mode=\"L\"` to `iio.imread()`."
```
The YAML parser of the lesson template stumbles over the quotes within quotes, so I added the backslash escape.

Reply:
Oh, sure! Thank you, @uschille.